generated/provider_dependencies.json

```
@@ -910,7 +910,7 @@
     "deps": [
       "apache-airflow>=2.9.0",
       "dnspython>=1.13.0",
-      "pymongo>=4.0.0,!=4.11,!=4.11.1"
+      "pymongo>=4.0.0,<=4.11"
```

```suggestion
      "pymongo>=4.0.0,<4.11"
```

Oops, I went too quick, we need to exclude 4.11 too.